my-school/src/actions/actions-portfolio.js

```diff
@@ -19,6 +20,19 @@ export const getAllActivitiesForUser = (id) => dispatch => {
         })
 }
 
+export const getAllActivities = () => dispatch => {
+    dispatch({type: FETCHING_ACTIVITIES})
+    axios.get(`https://my-school-v1.herokuapp.com/api/activities`)
```

Comment:
Yeah is this grabbing literally every single activity? I think that would re-bloat our bandwidth/memory usage. its fine with small data sets but it will be crushing at any kind of scale